reference/3.0/Microsoft.PowerShell.Management/Start-Process.md

```diff
@@ -364,25 +379,35 @@ Aliases:
 
 Required: False
 Position: Named
-Default value: Current directory
+Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
 ## INPUTS
 
 ### None
-You cannot pipe input to Start-Process.
+
+You cannot pipe input to this cmdlet.
+
 ## OUTPUTS
 
-### None or System.Diagnostics.Process
-When you use the PassThru parameter, Start-Process generates a System.Diagnostics.Process.
-Otherwise, this cmdlet does not return any output.
+### None, System.Diagnostics.Process
+
+This cmdlet generates a **System.Diagnostics.Process** object, if you specify the **PassThru**
+parameter. Otherwise, this cmdlet does not return any output.
 
 ## NOTES
-* This cmdlet is implemented by using the Start method of the System.Diagnostics,Process class. For more information about this method, see [Process.Start Method](https://msdn.microsoft.com/library/system.diagnostics.process.start) in the MSDN library.
+
+* This cmdlet is implemented by using the **Start** method of the **System.Diagnostics.Process**
+  class. For more information about this method, see
+  [Process.Start Method](/dotnet/api/system.diagnostics.process.start?view=netframework-4.7.2#overloads).
```

Comment:
Any reason deep linking to Overloads and not just the page in general?  Also, do we want to link to 4.7.2 specifically?

Reply:
The original MSDN link redirected there. I just copied the URL from the browser.